source/operations/concepts/availability-and-resiliency.rst

```diff
@@ -0,0 +1,153 @@
+   For example, if exactly half the drives in the erasure set become isolated due to a network fault, MinIO would consider quorum lost as it cannot establish a N+1 group of drives for the write operation.
+
+   .. figure:: /images/availability/availability-erasure-sharding-split-brain.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of erasure set where half the drives have failed
+
+      This node has 50% drive failure.
+      If parity is ``EC:8``, this erasure set cannot meet write quorum and MinIO rejects write operations to that set.
+      Since the erasure set still maintains read quorum, read operations to existing objects can still succeed.
+
+An erasure set which loses more drives than the configured parity has suffered data loss. 
+   For maximum parity configurations, the erasure set goes into "read only" mode if drive loss equals parity.
+   For the maximum erasure set size of 16 and maximum parity of 8, this would require the loss of 9 drives for data loss to occur.
+
+   .. figure:: /images/availability/availability-erasure-sharding-degraded-set.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of completely degraded erasure set
+
+      This erasure set has lost more drives than the configured parity of ``EC:4`` and has therefore lost both read and write quorum.
+      MinIO cannot recover any data stored on this erasure set.
+
+MinIO further mitigates the risk of erasure set failure by "striping" erasure set drives across each node in the pool.
+   MinIO automatically calculates the optimal erasure set size based on the number of nodes and drives.
+   It then selects one drive per node going across the pool for each erasure set, circling around if the erasure set stripe size is greater than the number of nodes.
+   This topology improves resiliency to the loss of a single node, or even a storage controller on that node.
+
+   .. figure:: /images/availability/availability-erasure-sharding-striped.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of a 16 node x 8 drive per node cluster, consisting of 8 16-drive erasure sets striped evenly across each node.
+
+      In this 16 x 8 deployment, MinIO would calculate 8 erasure sets of 16 drives each.
+      It allocates one drive per node across the available nodes to fill each erasure set.
+      If there were 8 nodes, MinIO would need to select 2 drives per node for each erasure set.
+   
+   In the above topology, the pool has 8 erasure sets of 16 drives each striped across 16 nodes.
+   Each node would have one drive allocated per erasure set.
+   While losing one node would technically result in the loss of 8 drives, each erasure set would only lose one drive each.
+   This maintains quorum despite the node downtime.
+
+Each erasure set is independent of all others in it's same pool.
```

Comment:
`it's` -> `its`
Maybe even `others in the same pool` if you like.

Reply:
Done